packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -25,6 +25,19 @@ const int _kMaxDroppedSwipePageForwardAnimationTime = 800; // Milliseconds.
 // user releases a page mid swipe.
 const int _kMaxPageBackAnimationTime = 300; // Milliseconds.
 
+/// Barrier color used for a barrier visible during transitions for Cupertino
+/// page routes.
+///
+/// This barrier color is only used for full-screen page routes with
+/// `fullscreenDialog: false`.
+///
+/// By default, `fullScreenDialog` Cupertino route transitions have no
```

Comment:
fullScreenDialog -> fullscreenDialog (here and elsewhere)

Reply:
Done 👍🏻